### ISSUE TITLE
value_error: ListFraudReviews200ResponseFraudReviewsInner.reasons 

### ISSUE THREAD

**Issue by beiping96** [tremendous-rewards/tremendous-python#41]
```
    return cls.__pydantic_validator__.validate_python(
pydantic_core._pydantic_core.ValidationError: 1 validation error for ListFraudReviews200ResponseFraudReviewsInner
reasons
  Value error, each list item must be one of ('Disallowed IP', 'Disallowed email', 'Disallowed country', 'Over reward dollar limit', 'Over reward count limit', 'VPN detected', 'Device related to multiple emails', 'Device or account related to multiple emails', 'IP on a Tremendous fraud list', 'Bank account on a Tremendous fraud list', 'Fingerprint on a Tremendous fraud list', 'Email on a Tremendous fraud list', 'Phone on a Tremendous fraud list', 'IP related to a blocked reward', 'Bank account related to a blocked reward', 'Fingerprint related to a blocked reward', 'Email related to a blocked reward', 'Phone related to a blocked reward', 'Allowed IP', 'Allowed email') 
[type=value_error, input_value=['IP related to a blocked...ed to a blocked reward'], input_type=list]
    For further information visit https://errors.pydantic.dev/2.9/v/value_error
```

I got this error with `tremendous-python = “^4.2.0”`
It seems this repo has a mismatch with the tremendous services side,
is it the tremendous service side respond an unexpected `reason`, or does this repo not have the newest `reason` enum?

**Comment by lucasmazza** [comment]
@beiping96 looks like our spec file is missing one of the potential values for `reason` - we will fix this on the server side, update this repo once the spec is up to date, and probably release a new package version right after that

**Comment by lucasmazza** [comment]
Fixed on version 4.3.0
